server/services/licenceSearchService.js

```diff
@@ -135,30 +156,38 @@ module.exports = function createLicenceSearchService(
       const bookingIds = licencesWithCOM.map((l) => l.booking_id)
       const prisoners = await prisonerSearchApi(systemToken).getPrisoners(bookingIds)
       const licencesAcc = []
-      const prisonerDecoratedLicences = licencesWithCOM.reduce(getPrisonerDecoratedLicences(prisoners), licencesAcc)
-      return getlicencesCSV(prisonerDecoratedLicences)
+      const decoratedLicences = licencesWithCOM.reduce(getPrisonerDecoratedLicences(prisoners), licencesAcc)
+      return getPrisonerDecoratedLicencesCSV(decoratedLicences)
     },
 
     async getLicencesRequiringComAssignment(username, prisonId) {
-      const systemToken = await signInService.getClientCredentialsTokens(username)
-      const licencesInStageWithAddressOrCasLocation = await licenceClient.getLicencesInStageWithAddressOrCasLocation(
-        'ELIGIBILITY',
-        systemToken
-      )
-      const bookingIds = licencesInStageWithAddressOrCasLocation.map((l) => l.booking_id)
-      const prisoners = await prisonerSearchApi(systemToken).getPrisoners(bookingIds)
-      const prisonersFilteredByPrisonCloseToHdced = prisoners.filter(
-        (p) => p.prisonId === prisonId && p.status !== 'INACTIVE OUT' && isCloseToHdced(p)
-      )
-      const offenderNumbers = prisonersFilteredByPrisonCloseToHdced.map((p) => p.prisonerNumber)
-      const probationDetails = await probationSearchApi(systemToken).getPersonProbationDetails(offenderNumbers)
+      const licences = await getEligibleLicencesWithAddressOrCasLocation(username)
```

Comment:
This file has grown a lot - It'd probably make sense to add a reports service.. this functionality isn't really anything to do with search